src/coreclr/jit/flowgraph.cpp

```diff
@@ -3887,12 +3887,6 @@ GenTree* Compiler::fgSetTreeSeq(GenTree* tree, bool isLIR)
             if (m_isLIR)
             {
                 node->ClearReverseOp();
-
-                // ARGPLACE nodes are not threaded into the LIR sequence.
```

Comment:
It's probably not worth re-running the CI for this, but the function header now has an outdated comment:
```
//    isLIR - whether the sequencing is being done for LIR. If so,
//            ARGPLACE nodes will not be threaded into the linear
//            order, and the GTF_REVERSE_OPS flag will be cleared
```

Reply:
There was another instance of this and I also took the opportunity to change some uses of `GetEarlyNode()` to `GetNode()`.